### DISCUSSION TITLE
Update generated repos

### DISCUSSION THREAD

**Discussion by hrvstr** [arthelokyo/astrowind#23] [General]
Hi, I generated a repo from this template last week and saw that there have been a lot of nice updates since then. Is there any way to keep my "fork" up to date with the development?

I tried adding this repo as a second remote and then fetching and merging it but it does not seem to work.
```
fatal: refusing to merge unrelated histories
```

**Comment by prototypa** [comment]
Hi @hrvstr

Since I created the template I've been thinking about what you're saying.

Due to the nature of Github Templates, what happens is that when a new repository is created based on a GitHub template, the code is downloaded and that's it. And that works well with very stable templates that serve as an initial guide without further update needs. But in this case, the template still needs a lot of work to bring it to production-ready levels like the ones I've worked with NextJs, for example, so there will be several more changes until it reaches version 1.0.0.

I have been dedicating the little extra time I have to work on the project, initially to understand Astro well and find the best practices that make it easy to adapt to the needs of the developers, but I certainly care a lot about being able to provide updates. When I finish some basic AstroWind tasks that I have pending, I will start to investigate what variants exist to solve your problem.

In the meantime check out this related Stack overflow post, it might help: https://stackoverflow.com/questions/56577184/github-pull-changes-from-a-template-repository

I will give you updates soon and if you find out about any solution, please let me know.

Thanks.

**Comment by hrvstr** [comment]
Okay, I guess there is no streamlined way to easily update themes in Astro. When I was using Hugo one could add a theme as a git submodule and easily update it later. I think for now I will just grab the new components and will redo my site from scratch once this template hits v1.0.

When I have some time on the weekend I will try some of the procedures from the stack overflow link but they seem rather experimental.

Thank you very much for your work and fast reply.

**Comment by MikeBinstock** [comment]
Thanks both for the conversation. I was thinking the same thing. I'll fork the project to get the new updates which are very valuable. Thank you for sharing this template, it looks great.
